### PR TITLE
Melodic migration

### PR DESCRIPTION
**Pull request by lvanasse**
Travail fait : Migration from Kinetic to Melodic

Est-ce que ça été testé et comment? : Oui, ça été testé sur le robot

Certaines lignes dans les launchs sont en commentaire, cela est intentionnel puisque ces nodes/configurations ne  sont pas encore fonctionnel.le.s.

### CODE REVIEW

**Review by island212 — APPROVED** [review]
Nice - Marc 2019

**Comment by IceSentry** [comment]
I would add the ROS version in the README for new devs or random people ending up here.

**Review by IceSentry — APPROVED** [review]

File: takin_bringup/package.xml [hunk 1]
```diff
@@ -14,12 +14,9 @@
   <exec_depend>teleop_twist_joy</exec_depend>
 
   <exec_depend>rgbd_launch</exec_depend>
-  <exec_depend>astra_camera</exec_depend>
```
Comment (LazyEngineerToBe) [inline takin_bringup/package.xml]: I don't think the astra packages should be removed from dependencies.

**Review by lit-af — DISMISSED** [review]

File: takin_bringup/package.xml (hunk 1 above)
Comment (Ludoenso) [reply takin_bringup/package.xml]: there's no more astra package in melodic.

Comment (LazyEngineerToBe) [reply takin_bringup/package.xml]: Okay

**Review by lit-af — APPROVED** [review]

**Comment by Ludoenso** [comment]
> I would add the ROS version in the README for new devs or random people ending up here.

@IceSentry Although I think this is a nice idea, this is out of scope from the PR. If you don't mind, it ought to be inside another PR updating the ReadMe documentation. Something similar to the capra_web_ui repo :).

**Pull request merged**